plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgresTaskFailureRecoveryTest.java

```diff
@@ -22,4 +25,22 @@ public TestPostgresTaskFailureRecoveryTest()
     {
         super(RetryPolicy.TASK);
     }
+
+    @Test
+    @Override
+    protected void testDeleteWithSubquery()
```

Comment:
It's surprising that the FTE at the TASK level runs successfully with the current implementation

Reply:
Should I copy the test from BaseFailureRecoveryTest to show here
@kokosing

Reply:
Feel better to show here and commented.

Reply:
it could be like

```
 assertThatThrownBy(() -> super.testDeleteWithSubquery())....
 ```